native/packages/react-native-bpk-component-button/src/utils.js

```diff
@@ -121,12 +132,17 @@ export const iconPropType = (props, propName, componentName) => {
   return false;
```

Comment:
Isn't `undefined` the normal value for custom prop types?

Reply:
Will check and change if necessary.

Reply:
Looks like airbnb's custom ones use `null`: https://github.com/airbnb/prop-types/blob/master/src/childrenOf.js

Reply:
I've updated the return type to be `?Error`, and it now returns `null` instead of `false` when there's no error.

Reply:
🙌